src/components/language-toggle/template.test.js

```diff
@@ -0,0 +1,120 @@
+const nunjucks = require("nunjucks");
+const fs = require("fs");
+const path = require("path");
+
+const { axe, toHaveNoViolations } = require("jest-axe");
+const { render } = require("../../utils/jestHelpers");
+
+expect.extend(toHaveNoViolations);
+
+const templatePath = "src/components/language-toggle";
+nunjucks.configure(path.dirname(templatePath), { autoescape: true });
+
+describe("oneloginLanguageSelect Component", () => {
+  const mockParams = {
+    ariaLabel: "test-aria",
+    activeLanguage: "en",
+    class: "test-class",
+    languages: [
+      {
+        code: "en",
+        text: "English"
+      },
+      {
+        code: "cy",
+        text: "Cymraeg"
+      }
+    ]
+  };
+
+  it("has the appropriate accessibility testing", async () => {
+    const renderedComponent = render(
+      "language-toggle",
+      "oneloginLanguageSelect",
+      mockParams
+    );
+
+    const results = await axe(renderedComponent.html());
+    expect(results).toHaveNoViolations();
+  });
+
+  it("renders the aria-label from params", () => {
+    const renderedComponent = render(
+      "language-toggle",
+      "oneloginLanguageSelect",
+      mockParams
+    );
+    const renderedNavElement = renderedComponent("nav");
+    expect(renderedNavElement.attr("class")).toContain("test-class");
+  });
+
+  it("renders the class from params", () => {
+    const renderedComponent = render(
+      "language-toggle",
+      "oneloginLanguageSelect",
+      mockParams
+    );
+    const renderedNavElement = renderedComponent("nav");
+    expect(renderedNavElement.attr("aria-label")).toBe("test-aria");
```

Comment:
The test definitions of this test and the previous one are the wrong way around

Reply:
Ah good point! Just amended and pushed!

Reply:
Grand! 😄